lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Jsou tady ty nástroje potřeba zmiňovat? Spíš by se tu hodilo vysvětlit, co je Schema, Table a Column a jak to odpovídá objektům z Pandas.</p><p> </p><p>A taky co je třeba VARCHAR(32)</p><p> </p><p>Použil jsi metodu ljust. Ta dělá co? (to je otázka, co může padnout, když to nějaká účastnice bude číst)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Nástroje jsem zmiňoval, aby absolvent kurzu měl přehled, co se používá v praxi, a nebyl až tak úplně v Python-Pandas bublině.</p><p> </p><p>Table, column, typy sloupců by to chtělo popsat.</p>

Reply:
<p>Nástroje bych zmínil. Dost se nám btw. osvědčuje DBeaver.</p><p> </p><p>Případně pgcli, mycli, ...</p>